assets/api-swagger/swagger.json

```diff
@@ -1970,6 +1970,184 @@
         }
       }
+    },
+    "/translations/requests": {
+      "get": {
+        "tags": [
+          "Translation Requests"
+        ],
+        "summary": "List translation requests.",
+        "parameters": [
+          {
+            "name": "request_id",
+            "in": "query",
+            "required": false,
+            "description": "The id of the translation request.",
+            "schema": {
+              "type": "integer"
+            }
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "required": false,
+            "description": "Get translation requests that has one of the following statuses.",
+            "schema": {
+              "type": "array",
+              "items": {
+                  "type": "string",
+                  "enum": [
+                      "created",
+                      "pending",
+                      "cancelled",
+                      "completed"
+                  ]
+              }
+            }
+          },
+          {
+            "name": "since_created",
+            "in": "query",
+            "required": false,
+            "description": "Get translation requests created since this timestamp, in ISO 8601 format.",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "since_updated",
+            "in": "query",
+            "required": false,
+            "description": "Get translation requests updated since this timestamp, in ISO 8601 format.",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            }
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "description": "The number of records to fetch. Maximum 50.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "default": 10
+            }
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "description": "The index of the first record to fetch.",
+            "required": false,
+            "schema": {
+              "type": "integer",
+              "default": 0
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List of translation requests.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/TranslationRequest"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid request."
+          },
+          "403": {
+            "description": "API key is not authorised to access this resource."
+          },
+          "500": {
+            "description": "An error occurred."
```

Comment:
How do these tie in with the actual responses we send from the service? Which one of them is shown on the API reference example calls?

Reply:
Our error responses are in different format than the rest of api, so we need to sort that out first. I left this as generic as can be until then. Will create a ticket for the task.